src/options.rs

```diff
@@ -122,7 +123,7 @@ impl ConfigOptions {
         let cdir = cargo_dir();
         ConfigOptions {
             crates_file: match matches.value_of("cargo-dir") {
-                Some(dirs) => (dirs.to_string(), fs::canonicalize(dirs).unwrap()),
+                Some(dir) => (format!("{}/.crates.toml", dir), fs::canonicalize(dir).unwrap().join(".crates.toml")),
```

Comment:
What the fuck are you talking about? Try it and see ™

I simply corrected the things to make the command line options work as intended. The comment you mentioned just look like copy-paste, and whoever would mark a variable named "..._file" as directory? Your code below also yields filename, by the way.